package.json

```diff
@@ -168,6 +179,11 @@
                     "command": "spring-boot-dashboard.localapp.debug-multiple",
                     "when": "view == spring-boot-dashboard && !viewItem",
                     "group": "action@12"
+                },
+                {
+                    "command": "spring-boot-dashboard.localapp.stop-multiple",
+                    "when": "view == spring-boot-dashboard && !viewItem",
+                    "group": "action@13"
```

Comment:
I wasn't very sure about the `group` item, as it doesn't seem to be used anywhere.

Reply:
> it doesn't seem to be used anywhere

It was contributing to context menu when you right click empty place inside this view. However it's not working now, which is probably caused by upstream changes in vscode.

Reply:
Got it, thanks for explaining!